Springboot-demo/src/main/java/com/hollszabolcs/demo/model/Employee.java

```diff
@@ -0,0 +1,26 @@
+package com.hollszabolcs.demo.model;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity(name = "employees")
+@Data
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
```

Comment:
Read what [@Data](https://projectlombok.org/features/Data) does exactly and you can remove all the redundant annotations.

Reply:
nothing has been changed here